requirements.txt

```diff
@@ -1,17 +1,17 @@
-boto3==1.2.4
+boto3==1.3.1
 cached-property==1.3.0
-click==6.2
+click==6.6
 coloredlogs==5.0
-GitPython==1.0.1
+GitPython==2.0.2
```

Comment:
For reference, the major version number bump is due to dropping Python 2.6 support